src/main.h

```diff
@@ -61,8 +61,8 @@ static const unsigned int MEMPOOL_HEIGHT = 0x7FFFFFFF;
 /** No amount larger than this (in satoshi) is valid */
 static const int64 MAX_MONEY = 2000000000 * COIN;
 inline bool MoneyRange(int64 nValue) { return (nValue >= 0 && nValue <= MAX_MONEY); }
-static const int64 MIN_TX_FEE = CENT;
-static const int64 MIN_RELAY_TX_FEE = CENT;
+static const int64 MIN_TX_FEE = CENT / 10;
+static const int64 MIN_RELAY_TX_FEE = CENT / 10;
```

Comment:
According to RFC 0007, the rate should stay at 1 cent / KB, not sure why this is needed to change then. It should be the actually calculation when rounding to KB that needs to be changed. See my comment there in CTransaction::GetMinFee()

Reply:
Still, if MIN_TX_FEE is redefined, every place in the protocol that MIN_TX_FEE is referenced must be protected by protocol upgrade protection IsProtocolV07(). For example:
main.h:static const int64 MIN_TXOUT_AMOUNT = MIN_TX_FEE;
this defines the minimum output amount. If left unprotected, it's a fork hazard.